custom.props

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Does it have to be named `custom.props`? That doesn't seem particularly specific to this scenario

Reply:
Unfortunately, it does.